pacta/build_with_tag.sh

```diff
@@ -1,51 +1,96 @@
 #! /bin/bash
 
-tag="${1:-latest}"
-clones="${2:-PACTA_analysis create_interactive_report StressTestingModelDev pacta-data}"
+# Examples:
+# # The tag is enforced
+# build_with_tag 0.0.0.999
+#
+# # Optionally give the names of the repos as trailing arguments
+# build_with_tag 0.0.0.999 PACTA_analysis StressTestingModelDev
+#
+# # With the help of pacta-find, you don't need to know the names
+# # of pacta siblings -- it's enough to know the path to the parent.
+# pacta_siblings="$(basename $(pacta-find ~/git))"
+# ./build_with_tag.sh 0.0.0.999 "$pacta_siblings"
```

Comment:
This example shows off the new ability to pass repos as trailing arguments -- which plays well with `pacta-find`.